cockatrice/src/messagelogwidget.cpp

```diff
@@ -359,69 +306,40 @@ void MessageLogWidget::logMulligan(Player *player, int number)
         return;
 
     if (number > -1) {
-        if (isFemale(player))
-            appendHtmlServerMessage(tr("%1 takes a mulligan to %n.", "female", number).arg(sanitizeHtml(player->getName())));
-        else
-            appendHtmlServerMessage(tr("%1 takes a mulligan to %n.", "male", number).arg(sanitizeHtml(player->getName())));
+        appendHtmlServerMessage(tr("%1 takes a mulligan to %n.", number).arg(sanitizeHtml(player->getName())));
```

Comment:
I actually, switched it slightly differently, which seemed to work and avoid the need for a null pointer.